src/stac_iitmandi/templates/stac_iitmandi/template.html

```diff
@@ -1,8 +0,0 @@
-{% extends 'stac_iitmandi/base.html' %}
-{% block content %}
-<div class="container bg-light mt-2">
-  <h1>Heading here</h1>
-  <p>This is dummy text Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-    consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-</div>
-{% endblock content%}
```

Comment:
Can you tell me a reason of deleting this file. In my opinion this should be there for future references.

Reply:
New issue will be created.